docs/next.config.ts

```diff
@@ -1,20 +1,38 @@
+import * as url from 'url';
+import * as path from 'path';
 import type { NextConfig } from 'next';
+// @ts-ignore
+// eslint-disable-next-line no-restricted-imports
+import withDocsInfra from '@mui/monorepo/docs/nextConfigDocsInfra.js';
 import { withPigment, extendTheme } from '@pigment-css/nextjs-plugin';
-import path from 'path';
+
 import { theme as baseTheme } from './src/theme';
+import rootPackage from '../package.json';
 
-const DATA_DIR = path.join(process.cwd(), 'data');
+const currentDirectory = url.fileURLToPath(new URL('.', import.meta.url));
+const DATA_DIR = path.join(currentDirectory, 'data');
 
 const nextConfig: NextConfig = {
   trailingSlash: false,
   env: {
     DATA_DIR,
+    CURRENT_VERSION: rootPackage.version,
```

Comment:
Where is this used?

Reply:
To be used in version dropdown (or span).